tests/Database/SelectFields/ComputedPropertiesTests/ComputedPropertiesTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rebing\GraphQL\Tests\Database\SelectFields\ComputedPropertiesTests;
+
+use Rebing\GraphQL\Tests\TestCaseDatabase;
+use Rebing\GraphQL\Tests\Support\Models\Post;
+use Rebing\GraphQL\Tests\Support\Models\User;
+use Rebing\GraphQL\Tests\Support\Traits\SqlAssertionTrait;
+
+class ComputedPropertiesTest extends TestCaseDatabase
+{
+    use SqlAssertionTrait;
+
+    public function testComputedProperty(): void
+    {
+        $user = factory(User::class)->create([
+        ]);
+
+        $post = factory(Post::class)->create([
+            'user_id' => $user->id,
+        ]);
+
+        $post->published_at = now();
+        $post->save();
+
+
+        $query = <<<'GRAQPHQL'
+{
+  users {
+    id
+    name
+    posts {
+      id
+      isPublished
+    }
+  }
+}
+GRAQPHQL;
+
+        $this->sqlCounterReset();
+
+        $result = $this->graphql($query);
+
+//         $this->assertSqlQueries(<<<'SQL'
+// select "users"."id", "users"."name" from "users";
+// select "posts"."id", "posts"."user_id" from "posts" where "posts"."user_id" in (?) order by "posts"."id" asc;
+// SQL
+//         );
```

Comment:
Btw, don't see a reason to not simple enable them; it's fine if they are not showing what is _really_ expected.

For now they're a snapshot of the current situation.

Reply:
yes, that was an oversight